packages/application-shell/src/components/navbar/navbar-skeleton.tsx

```diff
@@ -0,0 +1,163 @@
+import type { ReactNode } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { designTokens } from '@commercetools-uikit/design-system';
+
+const NavBarLayout = (
+  props: TNavBarSkeletonProps & { children: ReactNode }
+) => {
+  return (
+    <nav
+      aria-busy={true}
```

Comment:
Adding `aria-busy` here will prevent the screen-readers from reading this part until the skeleton is not rendered anymore. That is why I'm also using styled divs instead of more descriptive markup.

Reply:
Awesome! 💯